src/style.css

```diff
@@ -0,0 +1,117 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf") format("ttf"),
+    url("/fonts/Avenir-Heavy.ttf") format("ttf");
+}
+
+.searchFormBig {
+  position: relative;
+}
+
+.searchFormSmall {
+  position: relative;
+}
+
+.inputBig {
+  border: 0;
+  outline: none;
+  line-height: 22px;
+  padding: 0 0 0 62px;
+
+  margin-top: 20px;
+  margin-bottom: 20px;
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  border-style: solid;
+  cursor: pointer;
+  background-color: #fff;
+  border-color: #e1e7ed;
+  border-radius: 4px;
+  border-width: 1px;
+  height: 70px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.inputBig::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+}
+
+.inputSmall {
+  border: 0;
+  outline: none;
+  line-height: 19px;
+  padding: 0 0 0 33px;
+
+  margin-top: 11px;
+  margin-bottom: 10px;
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  border-style: solid;
+  cursor: pointer;
+  background-color: #fff;
+  border-color: #e1e7ed;
+  border-radius: 4px;
+  border-width: 1px;
+  height: 42px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.inputSmall::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #3d4e61;
+  font-size: 14px;
+}
+
+.imageBig {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  top: 24px;
+  left: 25px;
+  z-index: 1;
+}
+
+.imageSmall {
+  width: 11px;
+  height: 11px;
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  z-index: 1;
+}
+
+.inputBig:hover {
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchFormSmall:hover {
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inputBig:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
+  font-weight: regular;
+}
+
+.inputSmall:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```

Comment:
Omg, you duplicate a lot of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item